app/forms/IpRightsRegistrationNumberFormProvider.scala

```diff
@@ -23,7 +23,11 @@ import play.api.data.validation.{Constraint, Invalid, Valid}
 
 class IpRightsRegistrationNumberFormProvider @Inject() extends Mappings {
 
-  val maxLength: Int                                                                      = 100
+  val maxLength: Int         = 100
+  val rejectXssChars: String = """^[^<>"&]*$"""
+
+  val regexErrorKey: String = "regex.error"
```

Comment:
Regex values added here but not to the form logic. Does the validation below need updating?

Reply:
I think I missed it by mistake, will add in the logic